torch/package/_file_structure_printer.py

```diff
@@ -0,0 +1,61 @@
+from typing import List, Dict
+import pathlib
+
+
+def _print_file_structure(
+    filename: str, records: List[str], include: str = "**", exclude: str = ""
+):
+    space = "    "
+    branch = "│   "
+    tee = "├── "
+    last = "└── "
+    start = "─── "
+
+    class Folder:
+        def __init__(self, name: str):
+            self.name = name
+            self.sub_folders: Dict[str, Folder] = {}
```

Comment:
do we actually need to track folders and files are separately? afaict, a folder with no elements looks exactly the same as a file in the tree right? So potentially we could have just `self.children` and `self.is_dir`, which follows the unix way of doing it a little more closely.